packages/experimental-app-router/package.json

```diff
@@ -2,25 +2,24 @@
   "name": "@faustwp/experimental-app-router",
   "version": "0.0.4",
   "description": "Experimental: A Faust package to support Next.js' App Router",
-  "main": "dist/cjs/index.js",
-  "module": "dist/mjs/index.js",
-  "types": "dist/cjs/index.d.ts",
+  "exports": {
+    ".": "./dist/index.js",
+    "./package.json": "./package.json"
```

Comment:
Package exports will come in handy once we start shipping `rsc` specific functionality vs `client` specific functions.